### PR TITLE
Supports copying neuroglancer URL

### PR DESCRIPTION
**Pull request by tingzhao**

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -317,6 +318,7 @@ export default class Neuroglancer extends React.Component {
       brainMapsClientId,
       eventBindingsToUpdate,
       callbacks,
+      ngServer,
```
Comment (neomorphic) [inline src/index.jsx:321]: @tingzhao this needs to be added to the propTypes

Comment (neomorphic) [reply src/index.jsx:321]: as does onSelectionDetailsStateChanged

**Review by neomorphic — CHANGES_REQUESTED** [review]
Please see the inline comments, also the let declarations on line 410 & 411 should probably be const.

File: src/index.jsx (hunk 1 above)
Comment (tingzhao) [reply src/index.jsx:321]: changes uploaded

**Pull request merged**